Bulldozer.CSV/SampleCSVs/26 - BenevolenceRequest.csv

```diff
@@ -0,0 +1,5 @@
+﻿BenevolenceRequestText,BenevolenceRequestDate,BenevolenceRequestId,BenevolenceRequestFirstName,BenevolenceRequestLastName,BenevolenceRequestEmail,BenevolenceRequestCreatedById,BenevolenceRequestCreatedDate,BenevolenceRequestRequestedById,BenevolenceRequestCaseWorkerId,BenevolenceRequestCellPhone,BenevolenceRequestHomePhone,BenevolenceRequestWorkPhone,BenevolenceRequestGovernmentId,BenevolenceRequestProvidedNextSteps,BenevolenceRequestStatus,BenevolenceRequestResultSummary,BenevolenceRequestAddress,BenevolenceRequestAddress2,BenevolenceRequestCity,BenevolenceRequestState,BenevolenceRequestZip,BenevolenceRequestCountry
+Requesting help paying rent,2020-04-09 00:00:00.000,2,Barney,Rubble,barneyr@bedrock.com,10477,2020-04-09 15:46:30.210,10444,NULL,,,,,,Approved,,NULL,NULL,NULL,NULL,NULL,NULL
```

Comment:
Can we test without null's and sanitize this csv without null?

Reply:
Also we need a BenevolenceResult.csv sample.

Reply:
Done and done.